pylint/lint/pylinter.py

```diff
@@ -733,8 +734,8 @@ def enable_fail_on_messages(self):
     def any_fail_on_issues(self):
         return (
             self.stats
-            and self.stats.get("by_msg") is not None
-            and any(x in self.fail_on_symbols for x in self.stats["by_msg"])
+            and len(self.stats.by_msg) != 0
```

Comment:
```suggestion
            and self.stats.by_msg
```
?

Reply:
😅

Reply:
Can be removed altogether, since we now known that `stats` will be initiliazed with `by_msg`.